src/main.rs

```diff
@@ -67,10 +67,11 @@ struct Args {
     #[clap(
         short = 'x',
         long = "lattice",
-        default_value = "default",
+        default_values_t = ["default".to_string()],
+        value_delimiter = ',',
         env = "WASMCLOUD_LATTICE"
     )]
-    lattice: String,
+    lattice: Vec<String>,
```

Comment:
```suggestion
    lattices: HashSet<String>,
```

supernit, but just to be clear in the code without changing clap

Reply:
Also, I think we can do this too to prevent duplicates

Reply:
Clap unfortunately won't let you do that https://github.com/clap-rs/clap/discussions/4291 without using a custom value parser. I _could_ go that direction, but it ends up being the same code regardless to convert from a list of strings into a HashSet

Reply:
Oh that's fine then, all good. I don't think a CLI converting a Vec into a hashset and back is going to be the end of the world